openvdb/openvdb/openvdb.cc

```diff
@@ -28,14 +28,19 @@
         #error ABI = 6 is deprecated, CMake option OPENVDB_USE_DEPRECATED_ABI_6 suppresses this error
     #endif
 #endif
+#ifndef OPENVDB_USE_DEPRECATED_ABI_7
+    #if OPENVDB_ABI_VERSION_NUMBER == 7
+        #error ABI = 7 is deprecated, CMake option OPENVDB_USE_DEPRECATED_ABI_7 suppresses this error
+    #endif
+#endif
```

Comment:
We're should remove the ABI6 branching right? Think we're dropping that in 9. Same in the .cc

Reply:
Our current deprecation policy is to leave ABI=6 in place until 9.1 when we would drop it?

https://github.com/AcademySoftwareFoundation/openvdb/blame/master/tsc/process/deprecation.md#L20